{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -10,3 +10,4 @@ include_trailing_comma = true
 use_parentheses = true
 force_grid_wrap = 0
 multi_line_output = 3
+skip = 'migrations'
```

Comment:
https://pycqa.github.io/isort/docs/configuration/options.html#skip
You might want `extend_skip` instead?